src/global/global.h

```diff
@@ -198,6 +198,9 @@ struct parameters {
 #ifdef DE
   int out_float32_GasEnergy = 0;
 #endif
+#ifdef STATIC_GRAV
+  int custom_grav;  // flag to set specific static gravity field
```

Comment:
This is fantastic @evazlimen! I am so excited that you got RT working again. Regarding the VL integrator, you should definitely go ahead and modify the appropriate function calls there (and as @bcaddy pointed out, I think that's why the builds that use VL are broken at the moment). I'll also add that we are planning to reset the default builds (hydro, gravity, etc) to use the VL integrator rather than simple in the future, so definitely good to keep it up to date with any upgrades you make.